tests/unit/v1/test_async_collection.py

```diff
@@ -249,47 +249,27 @@ async def test_list_documents_w_page_size(self):
     @mock.patch("google.cloud.firestore_v1.async_query.AsyncQuery", autospec=True)
     @pytest.mark.asyncio
     async def test_get(self, query_class):
-        import warnings
-
-        query_class.return_value.stream.return_value = AsyncIter(range(3))
-
         collection = self._make_one("collection")
-        with warnings.catch_warnings(record=True) as warned:
-            get_response = collection.get()
-
-            async for _ in get_response:
-                pass
+        get_response = await collection.get()
```

Comment:
how does this map to the sync version?

Reply:
Same functionality, now neither are Iterator/Generator/AsyncGenerators. Both aggregate results so they can be reversed and return a list, though the async function is a coroutine that has that functionality.